images/linux/scripts/installers/java-tools.sh

```diff
@@ -85,6 +90,22 @@ for cmd in gradle java javac mvn ant; do
     fi
 done
 
+javaVersion=`java -version 2>&1 | head -n 1 | cut -d\" -f 2`
```

Comment:
use ` |&`  instead of `2>&1` as we do at the doc section

Reply:
Done.